.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -270,21 +287,86 @@ jobs:
           }'
           wait-for-completion: false
 
+  notify-slack-on-success:
```

Comment:
Unfortunately I could find no simpler way besides separate jobs to print different information on workflow success vs. failure. Happy to get advice from folks…. I tried SO MANY things.

Reply:
Finally got both success and failure notifications (at the correct times):

<img width="522" alt="image" src="https://github.com/broadinstitute/rawls/assets/484484/62eac26d-3521-4d97-9da3-7d6ee69e5a47">

Reply:
Yeah, it's not fun.  The only way I could find was a bash scripting step that rendered the needed conditional pieces like [this](https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/create-azure-billing-project.yaml#L73). That way you at least don't have to duplicate so much of the slack msg payload.